stl/inc/ranges

```diff
@@ -30,6 +30,191 @@ namespace ranges {
     concept viewable_range = range<_Rng>
         && (borrowed_range<_Rng> || view<remove_cvref_t<_Rng>>);
 
+    template <class _Rng>
+    concept _Simple_view = view<_Rng> && range<const _Rng>

```

Comment:
Is this used somewhere? I could not find any other occurence.

Reply:
It's not used here; I grabbed a bit too much moving the `_Pipe` machinery into this PR. I didn't bother to pull it out again since it's small, easily validated against the definition in [[range.utility.helpers]](http://eel.is/c++draft/range.utility.helpers), and used by more than one library component (it's a PITA having machinery in flight at the same time as multiple components that need it). 

That said, shout if you're disturbed by having it lie fallow here for a few days and I'll yank it.